src/UI/Component/Glyph/Factory.php

```diff
@@ -0,0 +1,482 @@
+	 *       2: >
+	 *          If clicking on the Back/Next GLYPH opens a previous view of an object, the Back Glyph MUST be used.
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be ‘Back'.
+	 * ---
+	 * @param	string|null	$action
+	 * @return 	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function back($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Next Glyph indicates a possible change of the view. The view change leads back to some previous view.
+	 *   composition: >
+	 *       The chevron-right glyphicon is used.
+	 *   effect: >
+	 *       The click on a Next Glyph opens a new view.
+	 *
+	 *
+	 * rules:
+	 *   usage:
+	 *       1: >
+	 *          Back and Next Buttons MUST be accompanied by the respective Back/Next Glyph.
+	 *   style:
+	 *       1: >
+	 *          If clicking on the Back/Next GLYPH opens a new view of an object, the Next Glyph MUST be used.
+	 *       2: >
+	 *          If clicking on the Back/Next GLYPH opens a previous view of an object, the Back Glyph MUST be used.
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be ‘Next'.
+	 * ---
+	 * @param	string|null	$action
+	 * @return 	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function next($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Sorting Glyphs indicate the sorting direction of a column in a table as ascending (up) or descending (down).
+	 *       It is a toggle reversing the ordering of a column.
+	 *   composition: >
+	 *       The Sort Ascending Glyph uses glyphicon-arrow-up.
+	 *   effect: >
+	 *       Clicking the Sort Ascending Glyph reverses the direction of ordering in a table.
+	 *
+	 * rules:
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be ‘Sort Ascending'.
+	 * ---
+	 * @param	string|null	$action
+	 * @return 	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function sortAscending($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Sorting Glyphs indicate the sorting direction of a column in a table as ascending (up) or descending (down).
+	 *       It is a toggle reversing the ordering of a column.
+	 *   composition: >
+	 *       The Sort Descending Glyph uses glyphicon-arrow-descending.
+	 *   effect: >
+	 *       Clicking the Sort Descending Glyph reverses the direction of ordering in a table.
+	 *
+	 * rules:
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be ‘Sort Descending'.
+	 * ---
+	 * @param	string|null	$action
+	 * @return 	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function sortDescending($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The User Glyph triggers the “Who is online?” Popover in the Top Navigation.
+	 *       The User Glyph indicates the number of pending contact requests and users online via the the Novelty Counter and Status Counter respectively.
+	 *   composition: >
```

Comment:
Indeed you are correct. The idea of the KS in regard to glyphs, is, the give them a clear context. This is not the same as enabling as much reuse as possible. If somebody feels the need to use this glyph in another context or if this glyphs needs a more flexible context in future, this is not an issue. It just needs a PR to this description and a decision of the JF. 

In this case, the aria-label would also need some adaption.